docs/api/file.md

```diff
@@ -13,7 +13,7 @@ if you upload something you have to store the cid to access is. (use data-protoc
 ## Upload
 
 ### `sendFile(File) => Promise<FileId>` 
```

Comment:
This should not be a header3, a bold text like **`sendFile(File) => Promise<FileId>`** would be better